frontend/src/bundles/auth/pages/auth.tsx

```diff
@@ -38,16 +38,18 @@ const Auth: React.FC = () => {
     );
 
     const getScreen = (screen: string): React.ReactNode => {
-        switch (screen) {
-            case AppRoute.SIGN_IN: {
+        const path = getAuthEndpointName(screen);
```

Comment:
if we don't have some subroutes, we can use `includes`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes?retiredLocale=uk

Reply:
Resolved in [ac58bca](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/113/commits/ac58bcaace5c25cf3dc956d96457febaa14b89da)